plugins/mumble_plugin_linux_x64.h

```diff
@@ -0,0 +1,14 @@
+// Copyright 2005-2016 The Mumble Developers. All rights reserved.
```

Comment:
Please call this file mumble_plugin_linux_64bit.h. It has no x86_64 specific content. Only the pointer size is relevant.